### ISSUE TITLE
alpine linux support in install_script

### ISSUE THREAD

**Issue by ermish** [DataDog/datadog-agent#2669]
Hey team,

tl;dr.
Add support for alpine linux in the install script. 

I am trying to install the datadog agent in my dockerfile which is based on alpine linux. 
I see there is a datadog agent base image, however I cannot switch my base image without massive changes. This is a large factor in switching all of my company's sites/services APM/logging management to Datadog.

**Comment by hkaj** [comment]
Hi @Ermish, thanks for reaching out.

Why are you installing the agent in your dockerfile? The recommended way to deploy the agent is to run one per host/VM, not one per container, so you should be able to run the agent container independently from your applications containers.
There are some cases where running the agent in the app container or pod/task is required, but they are very few.

Please let us know what your environment is like so we can discuss what works best for you. If you'd rather do that outside a github repo, feel free to reach out to [our support](https://docs.datadoghq.com/help/).

**Comment by Ermish** [comment]
@hkaj bump

**Comment by hkaj** [comment]
Sorry for the delayed response @Ermish 
Have you had a chance to look at our [documentation about Beanstalk](https://docs.datadoghq.com/integrations/amazon_elasticbeanstalk/#datadog-container-agent-configuration)? We recommend running the agent as a separate container.

**Comment by juanmrad** [comment]
I have the same need as shown above. I am running on GCP Google App Engine Flexible environment. which only takes my Dockerfile running and I want the agent to run on the same container image as my environment that way it can catch all my logs and data. I have no other way to run the agent and connect it to my instances.

**Comment by DevBOFH** [comment]
+1 I'm running containers on Fargate and need to pull in process metrics. The only option here is to run an agent inside the container.

**Comment by vadim-nilov** [comment]
@hkaj, there is another case if I want to collect logs from the web service which is located in the separate service container as well as collect app logs that is storing its logs inside the service container file system.
As far as I understood there is no simple way to collect all of these logs using the Agent in the separate service container

**Comment by frankpaul142** [comment]
> [@hkaj](https://github.com/hkaj), there is another case if I want to collect logs from the web service which is located in the separate service container as well as collect app logs that is storing its logs inside the service container file system. As far as I understood there is no simple way to collect all of these logs using the Agent in the separate service container

I agree with this, we need exactly the same thing and looks like there's no way to run the install script for alpine.

**Comment by sgammon** [comment]
Bump

**Comment by chouetz** [comment]
Hi @sgammon,

Thank you for taking the time to share your feedback. We recently introduced a system to help manage long-standing issues through automated stale management. We understand this may have come as a surprise for some existing issues. We want to apologize if it felt that way. It was not our intent, and we want to reassure you that every issue is important to us and reviewed carefully. For your specific issue, we’re reviewing it to ensure the discussion progresses and receives the right level of attention. Thank you for helping improve the Datadog Agent.


— The Datadog Agent Team

**Comment by levan-m** [comment]
Hello, 
Install script isn't really an issue. Datadog Agent isn't [supported](https://docs.datadoghq.com/agent/supported_platforms/?tab=linux) on Alpine and no plan to support it.
In Fargate case our recommendation is to run Agent as a sidecar, please see these docs for [EKS](https://docs.datadoghq.com/integrations/eks_fargate/?tab=admissioncontrollerdatadogoperator#installation) and [ECS](https://docs.datadoghq.com/integrations/aws-fargate/?tab=webui) for details.
